website/docs/d/nsxt_edgegateway.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_nsxt_edgegateway"
+sidebar_current: "docs-vcd-data-source-nsxt-edge-gateway"
+description: |-
+  Provides a VMware Cloud Director NSX-T edge gateway data source. This can be used to read NSX-T edge gateway configurations.
+---
+
+# vcd\_nsxt\_edgegateway
+
+Provides a VMware Cloud Director NSX-T edge gateway data source. This can be used to read NSX-T edge gateway configurations.
+
+~> **Note:** This datasource uses new VMware Cloud Director
```

Comment:
No need for a yellow note. Better a blue note.

Reply:
Changed.